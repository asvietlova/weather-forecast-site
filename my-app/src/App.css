.App {
  text-align: center;
}
.App-header {
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  font-weight: bold;
  color: rgb(48, 95, 197);
  height: 100px;
}
.days {
  height: fit-content;
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  gap: 5px;
  align-items: center;
  justify-content: center;
  font-size: calc(7px + 2vmin);
  color: rgb(61, 58, 58);
}
.day {
  max-height: 100%;
  width: 250px;
  border: 2px solid rgb(63, 103, 214);
  border-radius: 15px;
}
.temperature {
  display: flex;
  flex-direction: row;
  justify-content: space-around;
}
.day img {
  width: 130px;
  height: 110px;
}
