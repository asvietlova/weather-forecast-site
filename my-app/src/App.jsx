import "./App.css";

import WEATHER_FOR_WEEK from "./config";
import WeekWeather from "./components/week";
import Header from "./components/header";

function App() {
  return (
    <div className="App">
      {Header()}
      {WeekWeather({ listOfWeatherPerDay: WEATHER_FOR_WEEK })}
    </div>
  );
}

export default App;
