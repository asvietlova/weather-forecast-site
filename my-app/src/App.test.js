import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import App from "./App";

Enzyme.configure({ adapter: new Adapter() });

describe("App", () => {
  test("should contain class App", () => {
    const wrapper = shallow(<App />);

    expect(wrapper.find(".App").exists()).toBe(true);
    expect(wrapper.find(".App").length).toBe(1);
  });

  test("should match snapshot", () => {
    const wrapper = shallow(<App />);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
