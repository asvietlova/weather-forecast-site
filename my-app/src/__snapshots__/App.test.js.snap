// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `"<div class=\\"App\\"><header class=\\"App-header\\"><div>Погода в Киеве на неделю</div></header><div class=\\"days\\"><div class=\\"day\\"><p>понедельник, 21 февраля 2022 г.</p><img src=\\"./img/clear.png\\" alt=\\"ясно\\"/><div class=\\"temperature\\"><div><p>мин.</p><p>+2°</p></div><div><p>макс.</p><p>+7°</p></div></div><p>Давление, мм: <span>740</span></p><p>Влажность, %: <span>85</span></p><p>Ветер, м/сек.: <span>2.7</span></p></div><div class=\\"day\\"><p>вторник, 22 февраля 2022 г.</p><img src=\\"./img/cloudy.png\\" alt=\\"облачно\\"/><div class=\\"temperature\\"><div><p>мин.</p><p>+3°</p></div><div><p>макс.</p><p>+6°</p></div></div><p>Давление, мм: <span>735</span></p><p>Влажность, %: <span>95</span></p><p>Ветер, м/сек.: <span>2.4</span></p></div><div class=\\"day\\"><p>среда, 23 февраля 2022 г.</p><img src=\\"./img/some-clouds.png\\" alt=\\"облачно\\"/><div class=\\"temperature\\"><div><p>мин.</p><p>0°</p></div><div><p>макс.</p><p>+2°</p></div></div><p>Давление, мм: <span>740</span></p><p>Влажность, %: <span>80</span></p><p>Ветер, м/сек.: <span>2.9</span></p></div><div class=\\"day\\"><p>четверг, 24 февраля 2022 г.</p><img src=\\"./img/rain.png\\" alt=\\"дождь\\"/><div class=\\"temperature\\"><div><p>мин.</p><p>-2°</p></div><div><p>макс.</p><p>+3°</p></div></div><p>Давление, мм: <span>754</span></p><p>Влажность, %: <span>55</span></p><p>Ветер, м/сек.: <span>4.0</span></p></div><div class=\\"day\\"><p>пятница, 25 февраля 2022 г.</p><img src=\\"./img/sun-snow.png\\" alt=\\"ясно, временами снег\\"/><div class=\\"temperature\\"><div><p>мин.</p><p>-1°</p></div><div><p>макс.</p><p>+1°</p></div></div><p>Давление, мм: <span>755</span></p><p>Влажность, %: <span>60</span></p><p>Ветер, м/сек.: <span>2.0</span></p></div><div class=\\"day\\"><p>суббота, 26 февраля 2022 г.</p><img src=\\"./img/some-clouds-2.png\\" alt=\\"облачно\\"/><div class=\\"temperature\\"><div><p>мин.</p><p>-4°</p></div><div><p>макс.</p><p>+2°</p></div></div><p>Давление, мм: <span>754</span></p><p>Влажность, %: <span>50</span></p><p>Ветер, м/сек.: <span>1.6</span></p></div><div class=\\"day\\"><p>воскресенье, 27 февраля 2022 г.</p><img src=\\"./img/snow.png\\" alt=\\"снег\\"/><div class=\\"temperature\\"><div><p>мин.</p><p>-2°</p></div><div><p>макс.</p><p>+3°</p></div></div><p>Давление, мм: <span>750</span></p><p>Влажность, %: <span>62</span></p><p>Ветер, м/сек.: <span>2.0</span></p></div></div></div>"`;
